tensorflow/lite/kernels/internal/reference/reduce.h

```diff
@@ -354,7 +354,7 @@ inline bool QuantizedMeanOrSum(const T* input_data, int32 input_zero_point,
   }
 
   // Calculate mean by dividing output_data by num of aggregated element.
-  U num_elements_in_axis = 1;
+  size_t num_elements_in_axis = 1;
```

Comment:
Thank you.
Could you change the line 361 std::numeric_limits<size_t>::max()?
Also the line 187 and 191?

Reply:
fixed